protos/storage/session/session.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package session;
```

Comment:
I am not sure you want this in `go-btfs-common` - these are internal pb files and should not exposed in other projects ... unless guard uses this? It's also harder to manage changes if you put it in `go-btfs-common` as reviews + tag releases + go.mod changes in go-btfs will slow the process down.

Reply:
I suggest putting it in go-btfs as internal pbs directly, unless other reasons.

Reply:
just used to put pb here.
got it.
will move to inside go-btfs.